Info.pm

```diff
@@ -652,7 +652,8 @@ sub _get_v1tag {
 				# Remove garbage and retry
 				# (string is truncated in the
 				# middle of a multibyte char?)
-				$value =~ s/(.)$//;
+				#$value =~ s/(.)$//; # error https://rt.cpan.org/Ticket/Display.html?id=101620
```

Comment:
Yes, I'm saying remove the commented out '$value =~ s/(.)$//;' as it provides no value. Leave the comment pointing to the RT issue.